src/ui/mainwindow.cpp

```diff
@@ -148,7 +149,7 @@ MainWindow::MainWindow(Core::Application *app, QWidget *parent) :
     connect(ui->actionCheckForUpdate, &QAction::triggered,
             m_application, &Core::Application::checkForUpdate);
     connect(ui->actionAboutZeal, &QAction::triggered, [this]() {
-        QScopedPointer<AboutDialog> dialog(new AboutDialog(this));
+        std::unique_ptr<AboutDialog> dialog(new AboutDialog(this));
```

Comment:
I am really curious, apart from the support for move semantics that we don't use, what's the actual benefit from switching from `QScopedPointer` to `std::unique_ptr`?